### PR TITLE
Adding help page. closes #234

### LINKED ISSUE #234: Add a Help page
Need copy for page, and the URL. Blocks #236

### PR DESCRIPTION
**Pull request by webinista**

### CODE REVIEW

File: app/views/web/help.html.erb [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+            	 viewBox="0 0 11.9 11.7" style="enable-background:new 0 0 11.9 11.7;" xml:space="preserve"><title>Windows 8 or higher icon</title>
+            <g>
+            	<path d="M4.9,5.6H0.4V1.8l4.6-0.6V5.6z M4.9,10.5L0.4,9.9V6.2h4.6V10.5z M11.5,5.6H5.4V1.1l6.1-0.8V5.6z M11.5,11.4l-6.1-0.8V6.2
+            		h6.1V11.4z"/>
+            </g>
+            </svg> icon) and the letter <kbd>R</kbd> on your keyboard.
+            A small dialogue box like the one shown below will open up.</p>
+
+            <p>Copy and paste the following into the box and click the &quot;Run&quot; button:
+            <a title="Click to copy" data-clipboard-text="////bvacofil.dva.va.gov/dmdi$/ro/roaccesscf.vbs" href="////bvacofil.dva.va.gov/dmdi$/ro/roaccesscf.vbs">//bvacofil.dva.va.gov/dmdi$/ro/roaccesscf.vbs</a>
```
Comment (plusjeff) [inline app/views/web/help.html.erb]: The slashes need to be back slashes `\`

Comment (plusjeff) [inline app/views/web/help.html.erb]: Not an href either, since it's not a link, but a Windows file path.

File: app/views/web/help.html.erb [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+            <p>
+            	Select VBMS paperless e-folder from the window that pops up then click certify.
+                <%= image_tag 'help-page/help-certify-appeal-paperless.png', {alt: 'Certify Appeal'} %>
+            </p>
+
+            <p>
+            The first time you use it you'll have to log in with your <strong>VA Username
+            and Password</strong> <strong><em>AND</em></strong> your <strong>VACOLS regional office</strong> login.
+
+            <%= image_tag 'help-page/help-page-login.png', {alt: 'Caseflow login screen'} %>
```
Comment (plusjeff) [inline app/views/web/help.html.erb]: The screenshot does not have _both_ logins in it. @paultag Could you provide @webinista with this?

File: app/views/web/help.html.erb [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+        enter a Certification Date.
+        	<%= image_tag 'help-page/help-certify-appeal-paperless.png', {alt: 'Certify Appeal'} %>
+        	</p>
+        	</dd>
+
+        	<dt>What new features are you working on?</dt>
+        	<dd>We heard from our
+        	early testers that generating and uploading letters is a big pain for
+        	VBA employees so we’re working on a way to make that easier
+        	<span class="emoji">🙂</span>.</dd>
```
Comment (plusjeff) [inline app/views/web/help.html.erb]: This emoji does not render in IE11

File: app/views/web/help.html.erb [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+        	<dt>Who made Caseflow?</dt>
+        	<dd>Caseflow was
+        	developed by the Board of Veteran Appeals and the Digital Service in
+        	collaboration with VBA and users like you.
+        	</dd>
```
Comment (plusjeff) [inline app/views/web/help.html.erb]: Love the PBS-style messaging :smile:

**Comment by plusjeff** [comment]
@webinista Can you add the link from #440 in here? Going to close that PR.

File: app/views/web/help.html.erb (hunk 2 above)
Comment (paultag) [inline app/views/web/help.html.erb]: The SSOi login page?

File: app/views/web/help.html.erb (hunk 3 above)
Comment (paultag) [inline app/views/web/help.html.erb]: awwwwwww

File: app/views/web/help.html.erb (hunk 2 above)
Comment (plusjeff) [inline app/views/web/help.html.erb]: @paultag Yep!

**Comment by plusjeff** [comment]
Checklist for completion:

- [x] @webinista - Help link 
- [ ] @webinista - Emoji removal
- [ ] @webinista - Remove link for Windows file path for VACOLS
- [ ] @paultag Dual login screen shot(s)

**Comment by plusjeff** [comment]
@webinista Sorry - merge conflict :cry: . Could you do this?

File: app/views/web/help.html.erb (hunk 2 above)
Comment (paultag) [inline app/views/web/help.html.erb]: image

Comment (plusjeff) [inline app/views/web/help.html.erb]: Thanks! @webinista here's the screenshot for the second login screen.

**Comment by plusjeff** [comment]
Tests out in IE11

**Pull request merged**